Makefile

```diff
@@ -17,6 +17,12 @@
 # Disable CGO so that we always generate static binaries:
 export CGO_ENABLED=0
 
+ifeq ($(OS),Windows_NT)
```

Comment:
Where does this `OS` variable come from? Not sure if all versions of Windows will be reported as `Windows_NT`. I'd suggest to use `go env GOOS` as you have Go available and it is certain that it will return `windows`:

```make
ifeq ($(shell go env GOOS),windows)
```

Reply:
Done, thank you